src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -608,7 +597,7 @@ private Artifact chooseBinaryBits() throws MojoExecutionException {
         } else if ("Solaris".equals(os) || "SunOS".equals(os)) {
             plat = "solaris";
         } else if ("Mac OS X".equals(os) || "Darwin".equals(os)) {
-            plat = "mac";
+            plat = isBelowMacOSX_10_8() ? "mac" : "osx";
```

Comment:
Isn't that in contrary to what Grzegorz wrote at https://sourceforge.net/p/launch4j/bugs/160/#95fd ?!

> `launch4j-3.9-workdir-mac.jar` and `launch4j-3.9-workdir-osx.jar`. osx is the 10.8 version.

But you're using the `mac` version for `isBelowMacOSX_10_8()` ?!

Reply:
I think the issue is that there is just one package for OSX
https://sourceforge.net/p/launch4j/bugs/160/#59fc